securedrop/source_app/utils.py

```diff
@@ -112,3 +113,26 @@ def normalize_timestamps(filesystem_id):
                 "Couldn't normalize submission "
                 "timestamps (touch exited with %d)" %
                 rc)
+
+
+def check_url_file(path, regexp):
```

Comment:
This is perhaps exceedingly nitpicky, since it's used immediately below, but `check_url_file` is a pretty generic name. Maybe change it to something more specific, like `check_onion_address_file` or `check_si_url_file`, or add a docstring?

Reply:
docstring it is.